Release 5.20.0 is broken

#### Environment details

  - OS: macOS 12.3.1
  - Node.js version: 16.15.0
  - npm version: 8.10.0
  - `@google-cloud/storage` version: 5.20.0

#### Steps to reproduce

  1. Download https://registry.npmjs.org/@google-cloud/storage/-/storage-5.20.0.tgz
  2. There is no JS code in the file...

```
Cannot find module '@google-cloud/storage' or its corresponding type declarations.
```

--- a/src/nodejs-common/util.ts
+++ b/src/nodejs-common/util.ts
@@ -29,7 +29,14 @@ import {Duplex, DuplexOptions, Readable, Transform, Writable} from 'stream';
 import {teenyRequest} from 'teeny-request';
 import {Interceptor} from './service-object';
 import * as uuid from 'uuid';
-const packageJson = require('../../package.json');
+let packageJson: ReturnType<JSON['parse']> = {};
+try {
+  // if requiring from 'build' (default)
+  packageJson = require('../../package.json');
+} catch (e) {
+  // if requiring directly from TypeScript context
+  packageJson = require('../package.json');
+}
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const duplexify: DuplexifyConstructor = require('duplexify');

--- a/src/nodejs-common/util.ts
+++ b/src/nodejs-common/util.ts
@@ -32,10 +32,10 @@ import * as uuid from 'uuid';
 let packageJson: ReturnType<JSON['parse']> = {};
 try {
   // if requiring from 'build' (default)
-  packageJson = require('../../package.json');
+  packageJson = require('../../../package.json');
 } catch (e) {
   // if requiring directly from TypeScript context
-  packageJson = require('../package.json');
+  packageJson = require('../../package.json');
 }
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires

--- a/src/nodejs-common/util.ts
+++ b/src/nodejs-common/util.ts
@@ -29,7 +29,7 @@ import {Duplex, DuplexOptions, Readable, Transform, Writable} from 'stream';
 import {teenyRequest} from 'teeny-request';
 import {Interceptor} from './service-object';
 import * as uuid from 'uuid';
-import * as packageJson from '../../package.json';
+const packageJson = require('../../package.json');
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const duplexify: DuplexifyConstructor = require('duplexify');

